biggerquery/dagbuilder.py

```diff
@@ -0,0 +1,127 @@
+import shutil
+from pathlib import Path
+from biggerquery import Config
+Path(__file__).parent.absolute()
+from datetime import datetime
+
+
+def clear_dags_output_dir(workdir: str):
+    dags_dir_path = get_dags_output_dir(workdir)
+
+    print ("clearing dags_output_dir", str(dags_dir_path.resolve()))
```

Comment:
I think we should use logger with some default, local configuration instead of prints.

Reply:
+1

Reply:
we have agreed to stay with print, adding logger config is pointless, because nobody will use it in the build script